tests/jni_global_ref_is_deleted.rs

```diff
@@ -26,18 +22,24 @@ use util::{attach_current_thread, unwrap};
 /// *To avoid race condition this test routine should remain in a separate binary file.*
 
 #[test]
-#[ignore]  // JVM 10+ just aborts the process.
+#[ignore] // JVM 10+ just aborts the process.
 pub fn global_ref_is_dropped() {
     const VALUE: jint = 42;
 
     let env = attach_current_thread();
 
     let global_obj = {
-        let local_ref = AutoLocal::new(&env, unwrap(&env, env.new_object(
-            "java/util/concurrent/atomic/AtomicInteger",
-            "(I)V",
-            &[JValue::from(VALUE)]
-        )));
+        let local_ref = AutoLocal::new(
+            &env,
+            unwrap(
+                &env,
+                env.new_object(
+                    "java/util/concurrent/atomic/AtomicInteger",
+                    "(I)V",
+                    &[JValue::from(VALUE)],
+                ),
```

Comment:
Why does it add commas after the last argument to the method?
It is not, say, an arbitrarily sized list of constants that you often extends (e.g., imports, annotation parameters in Java) with no changes to adjacent lines. If you change the list of method parameters, you can _expect_ modified lines around.

Reply:
Personally I like current behavior: if you add or remove one parameter, then only one line is added or removed.

Reply:
> Why does it add commas after the last argument to the method?

It is about formatting the [multiline call](https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/expressions.md#multi-line-calls)

Reply:
> Personally I like current behavior: if you add or remove one parameter, then only one line is added or removed.

Just to make it clear: "current" -> new format from this PR, correct? :)

Reply:
@skletsun Right.